docs/go/activities.md

```diff
@@ -257,6 +257,17 @@ Activities to complete by calling the `Get()` methods of the future at a later t
 
 To implement more complex wait conditions on returned future objects, use `workflow.Selector`. Learn more on the [Go SDK Selectors](https://docs.temporal.io/docs/go/selectors) page.
 
+### Sharing Dependencies between Activities
```

Comment:
This is already documented here: https://docs.temporal.io/docs/content/how-to-develop-an-activity-definition-in-go/

I would prefer to deprecate this page, rather than add to it.
Or, reuse the content from above.

Reply:
👋 I was [asking](https://temporalio.slack.com/archives/CTDTU3J4T/p1632923907136400) about this in slack this morning - the confusion (at least from my perspective) is that it's not clear from the wording linked above in what scenarios using a struct is advantageous.  I think something like:

> Activities written as struct methods can use shared struct variables _such as connection pools to external services initialized when worker processes are started_.

would do the trick.

Reply:
Thanks for the feedback @asonawalla